### ISSUE TITLE
environment auto approve function not working for maker app

### ISSUE THREAD

**Issue by KrishanKumar1999** [microsoft/coe-starter-kit#8437]
### Does this question already exist in our backlog?

- [X] I have checked and confirm this is a new question.

### What is your question?

Issue: After creating  an environment through Maker app the environments are showing in Pending state. I want to get it auto approved.
 
Version of COE: Its happening after migrating to 4.29.

### What solution are you experiencing the issue with?

None

### What solution version are you using?

4.29

### What method are you using to get inventory and telemetry?

None

AB#3510

**Comment by Jenefer-Monroe** [comment]
I'm unsure what you mean by auto-approve. Can you please clarify? To my knowledge you have to explicitly approve these through the **_CoE Admin Environment Request_** app

**Comment by KrishanKumar1999** [comment]
You are correct but there is also an auto-approve feature by which we need not to manually approve it in CoE Admin Environment Request app. I was previously using it but now after the update of COE starter toolkit it stopped working.

**Comment by Jenefer-Monroe** [comment]
Can you please share a screenshot of where this settings is located? And then I'll send off to area owner.

**Comment by iamaninda93** [comment]
@Jenefer-Monroe 
Here is the screenshot. We are using "Maker Environment" app to create an environment. After providing the necessary details in Environment maker app we are seeing the environment request is in pending state. After manual approval of this environment an environment gets created.

What we want: After providing necessary details in the maker environment app then the environment should get auto approved rather than manually approved by Admin.


![MicrosoftTeams-image](https://github.com/microsoft/coe-starter-kit/assets/38653460/45c731e4-c816-4e5f-b0b7-32afcaa6f50f)

**Comment by iamaninda93** [comment]
@Jenefer-Monroe 
I have checked Youtube videos and found below settings might be helpful. Even after changing it “Yes” nothing helps.
Auto approve of environments were working before the upgrade of CoE starter kit April version. 
![IMG_6345](https://github.com/microsoft/coe-starter-kit/assets/38653460/18161024-88d0-4618-bbfb-0166b8b47646)

**Comment by iamaninda93** [comment]
I have found a similar post here.

https://github.com/microsoft/coe-starter-kit/issues/6556

**Comment by Jenefer-Monroe** [comment]
OK Thank you, not a new feature then thats what I was trying to assess. 
To dev owner for investigation.

**Comment by iamaninda93** [comment]
@Jenefer-Monroe 
Any update on the issue?

**Comment by Jenefer-Monroe** [comment]
Hello, no I do not have an update. We have assigned to the area owner and they will investigate.

**Comment by iamaninda93** [comment]
Hi @Jenefer-Monroe 

is there any update on the issue?

**Comment by gulzarudheen** [comment]
@iamaninda93 Started investigating on it.

**Comment by iamaninda93** [comment]
@gulzarudheen

Any update?

**Comment by harini-2-y** [comment]
We haven’t heard back in a while, so we’ll proceed to close this issue for now. If the issue persists or you need further assistance, feel free to log a new issue. We’re happy to help anytime. Thanks again for your engagement!

**Comment by harini-2-y** [comment]
Dear User,

We noticed that this issue has been pending for quite some time, and we have not received any recent updates from your end. In the absence of further communication, we are proceeding to close this ticket, assuming that your concerns have been addressed or resolved.
If you continue to experience any difficulties related to this matter, please feel free to open a new support ticket. This will help us ensure prompt and effective assistance, rather than reopening the new Issue.
Thank you for your understanding and cooperation.

Best Regards,
CoE Starter Kit Team
